src/main/resources/migrations/00001_create_table_event.sql

```diff
@@ -2,6 +2,19 @@
 
 --changeset uk.gov.pay:build event table
 CREATE TABLE event (
-    id VARCHAR(255) PRIMARY KEY
+    id BIGSERIAL PRIMARY KEY,
```

Comment:
Event structure envisioned here is different from https://github.com/alphagov/pay-architecture/blob/master/reporting/domain_model.md

Reply:
suggestion after discussion with @mrlumbu @kbottla @SandorArpa 

```
CREATE TABLE event (
    id BIGSERIAL PRIMARY KEY,
    sqs_message_id VARCHAR(255) NOT NULL,
    resource_type INT FOREIGN KEY RESOURCES(ID),
    resource_external_id VARCHAR(255),
    event_date TIMESTAMP WITH TIME ZONE DEFAULT (now() AT TIME ZONE 'utc') NOT NULL
    event_type VARCHAR(255),
    event_data jsonb not null,
);
```